### ISSUE TITLE
Accessible version perhaps shouldn't use JavaScript jump menu

### ISSUE THREAD

**Issue by prowlett** [chirun-ncl/chirun#19]
The accessible version has a menu for switch theme which uses JavaScript with an `onchange`. This is bad for keyboard navigation. I think this should be a list with a `submit` button to activate it, or just a `ul` list of options.

**Comment by georgestagg** [comment]
The old "accessible" theme has now been deleted. The default theme can now do the kinds of things it was originally used for via customisation.
